catalog-spec/catalog-spec.md

```diff
@@ -119,11 +119,56 @@ details on the two types and how you might use them see the [Static and Dynamic
 | id           | string        | **REQUIRED.** Identifier for the catalog.                    |
 | title        | string        | A short descriptive one-line title for the catalog.          |
 | description  | string        | **REQUIRED.** Detailed multi-line description to fully explain the catalog. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| summaries    | Map<string, [*]|Range Object> | A map of property summaries, either a set of values or a range. |
```

Comment:
Typo here, missing the open bracket around 'Range Object'

Reply:
Fixed.